flow/connectors/mysql/qvalue_convert.go

```diff
@@ -396,6 +409,13 @@ func QValueFromMysqlRowEvent(mytype byte, qkind qvalue.QValueKind, val any) (qva
 		return qvalue.QValueNull(qvalue.QValueKindJSON), nil
 	case []byte:
 		switch qkind {
+		case qvalue.QValueKindUInt64: // bit
+			var bit uint64
+			for _, b := range val {
+				bit = (bit << 8) | uint64(b)
+			}
+			slog.Warn("QQQcdc", slog.Uint64("val", bit), slog.Any("bits", val))
```

Comment:
TODO

Reply:
turns out qrep gets `bit` as `[]byte`, but cdc gets it as `int64`, which explains my confusion..

so removed this